packages/client/src/rpc/modules/engine.ts

```diff
@@ -806,6 +806,21 @@ export class Engine {
           message: 'ExecutionPayloadV2 MUST be used after Shanghai is activated',
         }
       }
+      const payloadAsV3 = params[0] as ExecutionPayloadV3
+      const { excessBlobGas, blobGasUsed } = payloadAsV3
+
+      if (excessBlobGas !== undefined && excessBlobGas !== null) {
+        throw {
+          code: INVALID_PARAMS,
+          message: 'Invalid PayloadV2: excessBlobGas is defined',
+        }
+      }
+      if (blobGasUsed !== undefined && excessBlobGas !== null) {
```

Comment:
```suggestion
      if (blobGasUsed !== undefined && blobGasUsed !== null) {
```
All the current pyspec tests pass with this change in in hive :D

Reply:
Thanks for this change :smile: Have just committed!